agent/structs/config_entry_gateways.go

```diff
@@ -283,6 +318,7 @@ type GatewayService struct {
 	Gateway     ServiceID
 	Service     ServiceID
 	GatewayKind ServiceKind
+	Port        int
```

Comment:
What would this be used for?

Reply:
Currently that's the service port (as set by the ingress gateway listener)